addon/templates/components/polaris-empty-search-result.hbs

```diff
@@ -0,0 +1,25 @@
+{{#polaris-stack
+  alignment="center"
+  vertical=true
+}}
+  {{#if withIllustration}}
+    {{!-- TODO: replace this with polaris-image when implemented. --}}
+    <img
+      src="empty-search.svg"
+      alt="Empty search results"
+      class="Polaris-EmptySearchResult__Image"
+    >
+  {{/if}}
+
+  {{polaris-display-text
+    size="small"
+    text=title
+  }}
+
+  {{#if description}}
+    {{polaris-text-style
+      variation="subdued"
+      text=description
```

Comment:
The description text should be [wrapped](https://github.com/Shopify/polaris-react/blob/master/src/components/EmptySearchResult/EmptySearchResult.tsx#L34) in a `p` element 😉